Multiple databases (draft) (#944)
neo4j/docs-aura: modules/ROOT/pages/managing-instances/instance-actions.adoc

From the instance card, select the more menu (*...*) then select *instance details*.
From the instance details you can rename an instance and view instance details.

See xref:managing-instances/instance-details.adoc[Instance details] for more information.

[.shadow]
.The more (...) menu on an AuraDB Free instance
image::instance-actions.png[]

> If the instance is set up to contain multiple databases, there is a *databases* tab where databases can be created. 

Reviewer: ```suggestion
If the instance is set up to contain multiple databases, you can create new databases from the *databases* tab. 
```

From the instance card, select the more menu (*...*) then select *instance details*.
From the instance details you can rename an instance and view instance details.

See xref:managing-instances/instance-details.adoc[Instance details] for more information.

[.shadow]
.The more (...) menu on an AuraDB Free instance
image::instance-actions.png[]

> If the instance is set up to contain multiple databases, you can create new databases from the *databases* tab. 

Author: ```suggestion
If the instance contains multiple databases, you can create new databases from the *databases* tab. 
```

Author: On second thoughts, you don't really need that there because you have "How to create a new database" in the actual multi-DB section.
